frame/support/test/Cargo.toml

```diff
@@ -25,6 +25,7 @@ pretty_assertions = "0.6.1"
 rustversion = "1.0.0"
 frame-metadata = { version = "13.0.0", default-features = false, path = "../../metadata" }
 frame-system = { version = "3.0.0", default-features = false, path = "../../system" }
+test-pallet = { package = "frame-support-test-pallet", default-features = false, path = "pallet" }
```

Comment:
maybe a simple comment saying that this pallet std feature is not activated on purpose to test construct_runtime errors.
```suggestion
# This pallet "std" feature is never activated on purpose, in order to test construct_runtime error message
test-pallet = { package = "frame-support-test-pallet", default-features = false, path = "pallet" }
```